browser/css/menubar.css

```diff
@@ -63,14 +63,60 @@
 	display: none;
 }
 .main-nav {
-	height: 32px; /* on mouseover menubar items, border emerges */
+	height: 36px; /* on mouseover menubar items, border emerges */
 	width: auto;
 	background: #ffffff;
-	padding: 3px 3px 3px 0;
+	padding: 0;
 	white-space: nowrap;
 	z-index: 12;
 	display: flex;
+	align-items: center;
+}
+
+nav:not(.hasnotebookbar) #main-menu-state:not(:checked) ~ #main-menu {
+	display: flex !important;
+}
+
+nav:not(.hasnotebookbar) #main-menu {
+	width: 100%;
+	height: auto;
+}
```

Comment:
Part of Vertical align nav bar and its child: no need to specifically target classic menu since #main-menu is only visible when using classic mode